app/src/components/fields/SingleDateField.tsx

```diff
@@ -16,6 +18,10 @@ interface IDateProps {
   other?: TextFieldProps;
 }
 
+const CalendarIcon: React.FC = () => {
```

Comment:
Nit: redundant

Reply:
Do you mean `React.FC`

Reply:
I meant that `<Icon />` should just be used twice inline, rather than using `const CalendarIcon`

Reply:
Took it out anyways, as it wasn't doing anything